scheduler/scheduler.go

```diff
@@ -16,14 +16,17 @@ import (
 )
 
 type TaskPlannerOptions struct {
-	ID               string
-	IsSecondaryQueue bool
-	StartedAt        time.Time
+	ID                   string
+	IsSecondaryQueue     bool
+	IncludesDependencies bool
+	StartedAt            time.Time
 }
 
 type TaskPlanner func(*distro.Distro, []task.Task, TaskPlannerOptions) ([]task.Task, error)
 
 func PrioritizeTasks(d *distro.Distro, tasks []task.Task, opts TaskPlannerOptions) ([]task.Task, error) {
+	opts.IncludesDependencies = d.DispatcherSettings.Version != evergreen.DispatcherVersionRevisedWithDependencies
```

Comment:
This means that `opts.IncludeDependencies` will be false if the dispatcher version isn't `DispatcherVersionRevisedWithDependencies`. That seems wrong.

Reply:
that is exactly correct

Reply:
But "with dependencies" implies "includes dependencies".

Reply:
every time I read you comment it still looks right/makes sense to me.

Reply:
oh